CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.2.2
+
+* `:vintage_net_wizard` 0.2.1
```

Comment:
To force `vintage_net_wizard` v0.2.1, the `mix.exs` dep needs to be updated. The `mix.lock` file doesn't get sent to  hex.pm, so changes in it won't affect users.

Reply:
Fixed